### DISCUSSION TITLE
Adding SkipGNN to PyGeo for better exploitation of protein-protein interaction networks

### DISCUSSION THREAD

**Discussion by THinnerichs** [pyg-team/pytorch_geometric#2689] [Ideas]
Hey Matthias,
I just wanted to request the addition of SkipGNN (https://www.nature.com/articles/s41598-020-77766-9) to the library. As published in Nature, I think it would be a great addition to PyGeo, as PPI networks make a substantial amount of of the use cases for bioinformatics (e.g. https://github.com/a-r-j/graphein).

The paper is well founded and quite good description of their methods and formulas. If needed I can try an implementation myself. 

Best wishes,

**Comment by rusty1s** [comment]
Thanks for the pointer. We can definitely try to add it to the library. Please feel free to make a first draft on this one and reach out to me in case you have any questions.
